main/res/drawable/ic_menu_pocket_query.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
When I created it AndroidStudio told me it will convert it for older platforms automatically. But I guess this is not good enough. We need it on gradle level, too.
I just searched a little bit and found this article: http://android-developers.blogspot.de/2016/02/android-support-library-232.html
It says that Android Support Library 23.2 can handle it, but we are still on 20.0. So at the moment it would be a problem.

I'll change it to png.